text/0108-reduce-traffic-resolved-ts.md

```diff
@@ -0,0 +1,114 @@
+## Summary
+
+The proposed method for reducing the traffic of pushing safe ts is hibernated region optimization. This method can be especially beneficial for large clusters since it can save a significant amount of traffic.
```

Comment:
> The proposed method for reducing the traffic of pushing safe ts is hibernated region optimization.

Currently, the main strategy in the new design is to minimize leader validation traffic and  `safe-ts` pushing traffic for valid leaders.